rvv-intrinsic-generator/rvv_intrinsic_gen/enums.py

```diff
@@ -169,19 +173,66 @@ def get(args,
           decorator,
           inst_type,
           mem_type=MemType.NO_MEM,
-          extra_attr=ExtraAttr.NO_ATTR):
+          extra_attr=ExtraAttr.NO_ATTR,
+          required_ext=None):
     if decorator is None:
       # vsetvl and vsetvlmax
-      return InstInfo(args["SEW"], args["LMUL"], args["OP"], inst_type,
-                      mem_type, extra_attr)
+      if required_ext:
```

Comment:
My python is pretty rusty. If required_exts is None, can we just pass it on to `InstInfo` without checking whether it is None or not?

Reply:
Yes, I think so.

Reply:
Updated and should be fine, thanks.